Bottom: 6656b3dabea0f1415107cbeea7b33d74ed35036c
Top:    5b15f5f7f4b37aba2e62a8a857486c6e41f1375f
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 18:38:34 +0530

Refresh of sched-core-add-manual

---

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index 841d78b6702f..0bda380d2845 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -2498,5 +2498,4 @@ static inline void membarrier_switch_mm(struct rq *rq,
 }
 #endif
 
-#define lenient_latency(attr) \
-	((attr)->latency_tolerance == MAX_LATENCY_TOLERANCE)
+#define lenient_latency(lat)	((lat) == MAX_LATENCY_TOLERANCE)
